### PR TITLE
LMB-133: Skeleton loaders

### PR DESCRIPTION
**Pull request by JonasVanHoof**
## Description

Instead of showing the three dots or loading text when we are waiting for our data to show up use skeletons so the user sees whats coming.

## How to test

1. Click through the app and see that everything works as expected + when you are waiting on data that it shows you a skeleton for the components to load.
2. You can also search on "loading" in the frontend project to make sure I didn't miss any.
3. Or search for "AuLoader" (here I let the auth frontend is what it is)

### CODE REVIEW

File: app/components/bestuursperiode-new-button.hbs [hunk 1]
```diff
@@ -2,6 +2,7 @@
   <AuButton {{on "click" this.openModal}}>
```
Comment (ruben-bottu) [inline app/components/bestuursperiode-new-button.hbs:2]: Is it possible that this component is not used anymore?

File: app/components/form/cancel-with-confirm.hbs [hunk 2]
```diff
@@ -1,5 +1,6 @@
 <AuButton @skin="secondary" {{on "click" this.cancel}}>{{yield}}</AuButton>
 
+<AuModalContainer />
```
Comment (ruben-bottu) [inline app/components/form/cancel-with-confirm.hbs:3]: Nice catch! Didn't know this was necessary. I think we mostly use the one in application.hbs

File: app/components/form/instance.js [hunk 3]
```diff
@@ -245,5 +246,8 @@ export default class InstanceComponent extends Component {
     };
     formStore.registerObserver(onFormUpdate);
     onFormUpdate();
+    await timeout(55);
```
Comment (ruben-bottu) [inline app/components/form/instance.js:249]: Seems like a magic number

**Review by ruben-bottu — APPROVED** [review]
Seems to work as expected. Haven't been able to see/find all skeleton components, but the ones I saw looked fine. Added some small comments.

File: app/components/form/cancel-with-confirm.hbs (hunk 2 above)
Comment (JonasVanHoof) [reply app/components/form/cancel-with-confirm.hbs:3]: It was very annoying adding these everywhere as the frontend keeps failing on it when it is not there 😓

File: app/components/bestuursperiode-new-button.hbs (hunk 1 above)
Comment (JonasVanHoof) [reply app/components/bestuursperiode-new-button.hbs:2]: I can't seem to find an implementation. Will remove it in a separate PR

File: app/components/form/instance.js (hunk 3 above)
Comment (JonasVanHoof) [reply app/components/form/instance.js]: Removed it! I added an await to this.registerObserver and does the same trick

**Pull request merged**